RemoteHealthcare/ConsoleGUI.cs

```diff
@@ -1,111 +1,149 @@
-                                    throw new Exception();
-                                }
-
-                                enteryValid = true;
-                            }
-                            catch (Exception)
-                            {
-                                Console.BackgroundColor = ConsoleColor.DarkRed;
-                                Console.WriteLine("Getal invoer is niet correct probeer opnieuw (Druk op een knop)");
-                                Console.BackgroundColor = ConsoleColor.Black;
-                                Console.ReadKey();
-                            }
-                        }
-
-                        Console.Clear();
-                        program.bikeManager.MakeConnectionAsync(serie, amountEntry).Wait();
+                    case (char)MenuOptions.READ_BIKE:
+                        ReadBike();
                         break;
-                    case "3":
-                        bool validEntery = false;
-                        int entryAmount = 0;
-                        while (!validEntery)
-                        {
-                            Console.Clear();
-
-                            Console.Write("Hoeveel data pakketen wil je ontvangen: ");
-                            try
-                            {
-                                entryAmount = Int32.Parse(Console.ReadLine());
-                                if (entryAmount! > 0)
-                                {
-                                    throw new Exception();
-                                }
-
-                                validEntery = true;
-                            }
-                            catch (Exception)
-                            {
-                                Console.BackgroundColor = ConsoleColor.DarkRed;
-                                Console.WriteLine("Getal invoer is niet correct probeer opnieuw (Druk op een knop)");
-                                Console.BackgroundColor = ConsoleColor.Black;
-                                Console.ReadKey();
-                            }
-                        }
-                        Console.Clear();
-                        program.hrManager.MakeConnection(entryAmount);
+                    case (char)MenuOptions.READ_HEARTHRATE:
+                        ReadHearthrate();
                         break;
-                    case "4":
-                        program.bikeManager.closeConnections();
-                        program.hrManager.closeConnections();
+                    case (char)MenuOptions.CLOSE_CONNECTION:
+                        CloseConnection();
                         break;
                 }
             }
 
             return Task.CompletedTask;
         }
 
+        private void CloseConnection()
+        {
+            this.program.bikeManager.CloseConnections();
+            this.program.hrManager.CloseConnections();
+        }
+
+        private void ReadHearthrate()
```

Comment:
Zie [Style Guide](https://github.com/Ti-2-1-RH-A1/remote-healthcare/blob/development/.github/STYLE_GUIDELINES.md#methodes) voor fout bij Methode structuur fout (Danwel hoe methode is opgebouwd of commentaar)